ocesql/ppout.c

```diff
@@ -1999,7 +1999,7 @@ void ppbuff_incfile(struct cb_exec_list *list){
 		}
 
 		memset(incmsg, 0, 256);
-		sprintf(incmsg, "%s incfile end:%s",INC__END__MARK , filename);
+		sprintf(incmsg, "%s incfile end:%-s",INC__END__MARK , filename);
```

Comment:
It seems that the update of the line 2002 does not solve warning messages.
https://github.com/yutaro-sakamoto/Open-COBOL-ESQL/runs/5878482122?check_suite_focus=true#step:8:217
```
ppout.c:2002:35: warning: ‘%-s’ directive writing up to 511 bytes into a region of size 171 [-Wformat-overflow=]
 2002 |   sprintf(incmsg, "%s incfile end:%-s",INC__END__MARK , filename);
      |                                   ^~~                   ~~~~~~~~
```
If the warning message seems difficult to solve, please undo the update.